src/lib.rs

```diff
@@ -239,7 +308,60 @@ impl Client {
         webdriver: &str,
         cap: webdriver::capabilities::Capabilities,
     ) -> Result<Self, error::NewSessionError> {
-        Session::with_capabilities(webdriver, cap).await
+        Session::<C>::with_capabilities(webdriver, cap)
+            .await
+    }
+
+}
+
+impl<C> Client<C>
+    where
+        C: connect::Connect + Unpin + 'static + Send + Sync + Clone,
+{
+    /// Create a new [`Client`][crate::Client] associated with a chosen HttpsConnector and a new WebDriver session on the server
+    /// at the given URL.
+    ///
+    ///
+    ///
+    /// Calls `with_capabilities_and_connector` with an empty capabilities list.
+    pub async fn new_with_connector(
+        webdriver: &str,
+        connector: C,
+    ) -> Result<Self, error::NewSessionError>
+    {
+        Self::with_capabilities_and_connector(
+            webdriver,
+            webdriver::capabilities::Capabilities::new(),
+            connector,
+        )
+            .await
+    }
+
+    /// Create a new `Client` associated with a chosen HttpsConnector and a new WebDriver session on the server
+    /// at the given URL.
+    ///
+    ///
```

Comment:
Let's get rid of all these empty doc lines all over the place. They aren't rendered in the documentation anyway, and they make the comments harder to read :)